content/en-us/resources/the-mystery-of-duvall-drive/developing-a-moving-world.md

```diff
@@ -167,7 +167,7 @@ local function LightningFunc(info)
 	…
 ```
 
-In addition, we run the sequence to set textures, positions, and brightness, run tweens, and use wait(number). Randomized parameters are from the info structure that we received from the server, and some numbers are fixed.
+In addition, we run the sequence to set textures, positions, and brightness, run tweens, and use `task.wait(number)`. Randomized parameters are from the info structure that we received from the server, and some numbers are fixed.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) and the `|` adjusts the display text (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
In addition, we run the sequence to set textures, positions, and brightness, run tweens, and use `Library.task.wait()|task.wait(number)`. Randomized parameters are from the info structure that we received from the server, and some numbers are fixed.
```